lighthouse-cli/test/smokehouse/smoke-test-dfns.js

```diff
@@ -69,11 +69,6 @@ const SMOKE_TEST_DFNS = [{
   expectations: 'perf/expectations.js',
   config: 'perf/perf-config.js',
   batch: 'perf-metric',
-}, {
```

Comment:
Currently, Smokerider uses `smoke-test-dfns` as a super easy way to get a full description of all the tests, modify expectations, runs, and then uses `{collateResults, report}` on the results.

Gotta figure out how to do that with your `*.smoke.js` changes ...

Reply:
I suppose I would import every `*.smoke.js`? Maybe we could have a function that does that?

Reply:
Actually, don't bother with that, I'll do that on my side. Not sure if that'll be the best approach yet.

Reply:
that's basically what https://github.com/GoogleChrome/lighthouse/pull/8962/files#diff-b7fef6a2e94d6698d40bf02b3fb67d49R17 is doing 

happy to make this a shared function in a separate file if it would help (will need to modify it to process argv for the filtering logic later anyhow)

Reply:
sure, let's do it